Solver_interface/include/CGAL/SCIP_mixed_integer_program_traits.h

```diff
@@ -25,6 +25,11 @@
 
 namespace CGAL {
 
+#if (_MSC_VER == 1500)
```

Comment:
Does not seem to be working as expected:
https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-5.6-Ic-269/Solver_interface_Examples/TestReport_Christo_MSVC2015-Release-64bits.gz